pxl_scripts/px/namespace/namespace.pxl

```diff
@@ -32,7 +32,11 @@ def pods_for_namespace(start_time: str, namespace: px.Namespace):
     df = px.DataFrame(table='process_stats', start_time=start_time)
     df = df[df.ctx['namespace'] == namespace]
     df.pod = df.ctx['pod_name']
-    df = df.groupby(['pod']).agg()
+    df = df.groupby(['pod']).agg(
+        rss_bytes=('rss_bytes', px.mean),
+        vsize_bytes=('vsize_bytes', px.mean),
```

Comment:
Since we show bytes now, we don't need to actually have it in the name.

Reply:
![image](https://user-images.githubusercontent.com/5456019/94721091-380f1180-030a-11eb-9777-9531743ea883.png)
Updated